Project.toml

```diff
@@ -0,0 +1,19 @@
+name = "ChainRulesTestUtils"
+uuid = "cdddcdb0-9152-4a09-a978-84456f9df70a"
+version = "0.1.0"
+
+[deps]
+ChainRules = "082447d4-558c-5d27-93f4-14fc19e9eca2"
```

Comment:
Do we need to have `ChainRules` as a dep? I would have thought that we would be able to get away with just having `ChainRulesCore` and `FiniteDifferencing` as deps?

Reply:
I think `ChainRules` can be a test-only dependency?